spec/traits.dd

```diff
@@ -1040,9 +1047,10 @@ void main()
     int b;
     static assert(!__traits(isSame, a => a + b, a => a + b));
 
-    // lambdas calling a function inside their body are considered incomparable
+    // lambdas calling other functions are comparable
     int f() { return 3;}
-    static assert(!__traits(isSame, a => a + f(), a => a + f()));
+    static assert(__traits(isSame, a => a + f(), a => a + f()));
+    test!(a => a + f())();
```

Comment:
Maybe you could inline `test` here, s.t. the reader has a consistent flow?

Reply:
The idea here is to show the construct works across scopes.